jina/peapods/runtimes/asyncio/rest/__init__.py

```diff
@@ -53,9 +53,19 @@ async def serve(self, sockets=None):
         from .....helper import extend_rest_interface
         import os
 
+        app_wrapper = AppWrapper(self.args, self.logger)
+        fastapi_app = app_wrapper.get_fastapi_app()
+        print(
+            '# extend_rest_interface.__code__.co_argcount',
+            extend_rest_interface.__code__.co_argcount,
+        )
+        if extend_rest_interface.__code__.co_argcount == 2:
+            fastapi_app = extend_rest_interface(fastapi_app, app_wrapper.send_request)
```

Comment:
How do we generalize that the 2nd arg would always be `send_request`?

Reply:
good question. I'm not so happy with the implementation I did there. It was just the only thing coming to my mind. I'm sure there must be a better solution.